app/models/submission.rb

```diff
@@ -16,7 +16,7 @@ class Submission < ActiveRecord::Base
   validates :english_keywords, presence: true
   has_many :authorships, dependent: :destroy
   has_many :article_revisions, dependent: :destroy
-  has_one :article
+  has_many :article
```

Comment:
To musi pozostać tak jak było. Trzeba będzie utworzyć nowe powiązanie - np. `follow_up`, które będzie pozwalało łączyć jeden artykuł z wieloma zgłoszeniami. Ale to jest odrębna relacja od tej tutaj.